src/gatsby-plugin-theme-ui/index.ts

```diff
@@ -0,0 +1,31 @@
+import _ from 'lodash'
+
+import preset from '@rebass/preset'
+import Colors from 'utils/colors'
+
+export const fonts = {
+  body: 'Lato, system-ui, sans-serif',
+  heading: 'Alegreya Sans',
+  monospace: 'Fira Mono',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)